backend/routes/Plugins.ts

```diff
@@ -23,17 +25,43 @@ const upload = multer();
 const logger = getLogger('Plugins');
 const getFiles = (req: Request<any, any, any, any, any>) => req.files as { [fieldname: string]: Express.Multer.File[] };
 
+const getYamlUrls = (yamlUrl?: string | string[]) => {
```

Comment:
Just to increase type safety, we can add a return type
```suggestion
const getYamlUrls = (yamlUrl?: string | string[]): string[] => {
```

Reply:
@Vorbert-Kruk it's already inferred to string [] type so there is no need for explicit type assignment

Reply:
Yeah, it is but as right now there are multiple points of return, was wondering if making the return type explicit would be a good idea

But as you've mentioned it's not something required, so I'm leaving the decision up to you 👍